internal/store/access_logs.go

```diff
@@ -83,10 +83,11 @@ func (s *Store) AccessLogsByClusterName(groupID, clusterName string, opts *atlas
 		if opts != nil {
 			if opts.Start != "" {
 				startTime, _ := time.Parse(time.RFC3339, opts.Start)
-				result = result.Start(startTime)
+				result = result.Start(startTime.Unix())
```

Comment:
Hey @gssbzn , I think I recall you mentioning a required change for the times at some point.

Is this what you were referring to?

Reply:
nop, I was asking at some point to double check if for those args which are now using the `time` package we could use flags native time support instead of strings and doing casting, I also know that for this API I think they ar enow int so the extra casting may not be correct anymore

Reply:
Ok cool I'll replace the double cast